wire-ios-sync-engine/Source/TeamRole+String.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireDataModel
+
+extension TeamRole {
+
+    static func stringValue(from intValue: Int) -> String {
+        if let role = TeamRole(rawValue: intValue) {
```

Comment:
This method makes little sense, and I see it's used as

```swift
TeamRole.stringValue(from: selfUser.teamRole.rawValue)
```

It would make more sense to have a computed property that returns a string:

```swift
extension TeamRole {

  var analyticsValue: String {
    switch self {
    case .partner:
      return "external"
    case .member, .admin, .owner:
      return "member"
    case .none:
      return "wireless"
    }
  }

}
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/6ae109e5f33791c38a322340a88a5e7cdfae446c